frontend/components/Footer/Footer.styl

```diff
@@ -1,7 +1,20 @@
 @import '../shared/mixins.styl';
 
+body {
+  margin: 0;
+  padding-bottom: 6em;
+}
+
 footer {
   padding: 2em 0;
+  height: 7em;
+  position: absolute;
```

Comment:
Should this be `fixed`? this breaks the layout on desktop:

![image](https://cloud.githubusercontent.com/assets/442115/24873898/9ac647a6-1df0-11e7-822b-1ae08281808e.png)